frontend/packages/metal3-plugin/src/components/host.tsx

```diff
@@ -0,0 +1,130 @@
+import * as React from 'react';
+import {
+  // BaremetalHostRole,
+  // BaremetalHostStatus,
+  // getHostMachineName,
+  getHostBmcAddress,
+  getName,
+  getNamespace,
```

Comment:
Same comment as in #1592, these aren't kubevirt specific. They should be added somewhere that can be used generally across console.

Reply:
I am happy to move this to console/shared in a subsequent PR and gradually reduce the dependency on web-ui-components. To unblock work depending on this PR. Would that be sufficient?

Reply:
We plan to move `kubevirt-web-ui-components` to `console/shared` or similar packages within one of the last steps of merging `web-ui`. Until this happens, we would need to maintain these utility functions (or similar code) on two places - `kubevirt-web-ui` is still in play until `openshift-console` meets full feature parity.

See my response in #1592 for more details.

Reply:
> I am happy to move this to console/shared in a subsequent PR and gradually reduce the dependency on web-ui-components.

The idea was that `@console/shared` contains code used by Console itself as well as other packages. The typical contribution workflow here would be moving code from `public` or `@console/app` to `@console/shared`. Package owners should be inherited from monorepo root `OWNERS`.

For code not needed by Console itself, we could add a dedicated package, e.g. `@console/rhhi-shared` (or similar). Common `kubevirt/web-ui` or `kubevirt-web-ui-components` code (KubeVirt, Metal3, Storage) could be moved there. Package owners should be declared explicitly, including people who work on said projects.

> We plan to move `kubevirt-web-ui-components` to `console/shared` or similar packages within one of the last steps of merging `web-ui`.

I'd create a dedicated package for that, e.g. `@console/rhhi-shared` suggested above.

(`kubevirt-web-ui-components` is not KubeVirt only, it also covers Metal3 and Storage.)

Reply:
I agree that for getting rid of kubevirt/web-ui-components dependency creating of a separate console/rhhi-shared package makes sense as it would contribute to speeding up the process.

On the other hand in ideal world it seems to me that package specific code can live in the package directly (e.g. metal3 specific components) and common ones should align to general use in console and therefore end up in console/shared. But I agree that we're not there yet and it will take time to get there, so the solution you propose is perfectly reasonable.

Regarding Sam's specific comment, these selectors can be used across the whole console codebase so I think it is quite reasonable to put it directly to console/shared.

Reply:
> package specific code can live in the package directly (e.g. metal3 specific components) and common ones should align to general use in console and therefore end up in console/shared

Yes, that's an ideal solution - `console-shared` changes should pass core team reviews, so any new code we pull from `kubevirt/web-ui` fork or `kubevirt-web-ui-components` library would be revisited and aligned with Console core.

We should aim for the best possible solution, so +1 on just going straight through `console-shared`, bit by bit.